openbas-model/src/main/java/io/openbas/database/model/InjectDependency.java

```diff
@@ -0,0 +1,52 @@
+package io.openbas.database.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import io.openbas.helper.MonoIdDeserializer;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.Instant;
+import java.util.Objects;
+
+@Setter
+@Getter
+@Entity
+@Table(name = "injects_dependencies")
+public class InjectDependency {
+
+    @EmbeddedId
+    @JsonProperty("dependency_relationship")
+    private InjectDependencyId compositeId = new InjectDependencyId();
+
+    @Column(name = "dependency_condition")
+    @JsonProperty("dependency_condition")
+    private String condition;
+
+    @CreationTimestamp
```

Comment:
Since we're using @UpdateTimestamp on the updateDate, seems logical to have @CreationTimestamp here. I guess creationDate = now(); would work too but it seemed more coherent to use @CreationTimestamp. Is it bothering you ?

Reply:
I prefer this annotation too and it will be great to see it more often !